6_visualize/src/prep_timeline_fun.R

```diff
@@ -0,0 +1,59 @@
+prep_timeline_fun <- function(DateTime, timestep_ind, timeline_config, spark_config, legend_text_cfg) {
+
+  this_DateTime <- as.POSIXct(DateTime, tz = "UTC") # WARNING, IF WE EVER MOVE FROM UTC elsewhere, this will be fragile/bad.
```

Comment:
are we displaying the UTC time or the local timezone time? Seems like the former, but don't viewers of the viz want to see the local tz?

Reply:
Good question. When [we showed the timestamp, we displayed that in the local timezone](https://github.com/USGS-VIZLAB/vizstorm-GIF/blob/master/6_visualize/src/prep_datetime_fun.R#L4) but UTC is what is being passed around throughout all of the frame creation. It would make sense to convert the dates to local time before displaying here, too.